location/src/main/java/edu/artic/location/FuzedLocationProvider.kt

```diff
@@ -113,9 +169,21 @@ class FuzedLocationProvider(private val context: Context) : LocationProvider {
         trackingRequested = false
         if (isTracking) {
             fuzedLocationClient.removeLocationUpdates(locationCallback)
+            mSensorManager.unregisterListener(this)
             isTracking = false
         }
     }
 
+    override fun onSensorChanged(event: SensorEvent) {
+        if (event.sensor.type == Sensor.TYPE_ACCELEROMETER) {
+            mGravity = event.values
+        } else if (event.sensor.type == Sensor.TYPE_MAGNETIC_FIELD) {
+            mGeomagnetic = event.values
+        }
+    }
+
+    override fun onAccuracyChanged(p0: Sensor?, p1: Int) {
```

Comment:
Please use same parameter names as defined in interface, replace body of method with ` = Unit`